### DISCUSSION TITLE
Can a machine be racist? Artificial Intelligence has shown troubling signs of bias, but there are reasons for optimism

### DISCUSSION THREAD

**Discussion by Aaron25564448** [1101-GS4719-C/Python-Community-GS4719-C#1152] [General]
AI究竟深入人們的生活到什麼程度，看完這篇文章就能理解了。用於預測犯罪的AI受到過去白人至上主義時期留下的犯罪紀錄而產生有些歧視、偏見的結果。確實，各種AI都號稱具有世界級的資料庫，但說真的就是過去社會所留下來的資訊，而其中又摻雜了過去社會的黑暗面。人類想利用藉由控制機器創造更好的社會，我們對這些理性客觀的數字的毫不質疑卻反過來控制人類的判斷。最佳的演算法究竟是否存在？是人在控制機器，還是機器控制著人？我認為人類要邁向下一個科技階段，我們所創造出的人工智慧究竟能否與人一樣學習資訊素養、避免使用具爭議性或是有偏見疑慮的資料會是一項重要的課題。
文章連結：
https://theconversation.com/can-a-machine-be-racist-artificial-intelligence-has-shown-troubling-signs-of-bias-but-there-are-reasons-for-optimism-197893

**Comment by KevinCheng42** [comment]
我覺得人工智慧的資料是由我們人類提供的，其中必存在一定程度的歧視與偏見，取決於資料的分析方式，如同下篇報導所述。
雖然人工智慧有其局限性，但我相信其仍可幫助我們解決一些複雜的社會問題，只要我們能夠注意和解決其中存在的偏見和不公正問題。

科技時代的迷思：大數據資料真的客觀嗎？
https://storystudio.tw/article/gushi/the-fuzzy-and-the-techie/

**Comment by YoYooHuang** [comment]
AI是人類開發出來的，其便利及實用性中必藏有許多隱憂，這都是值得我們去探討的。它所有參考知識來源都是來自於人類給予的訓練素材，他所謂的"辨識能力"也是由人類給他的資訊所決定。這也代表著AI不是自己產生錯誤偏見或不公正，而這一切都是人類創造出來的。AI就像一把大刀，能做出一桌好菜，也能大肆殺人。如何正確的訓練及使用AI才是最重要的課題。

**Comment by curtis1017** [comment]
如何使用AI這個議題確實重要，各種事物可以帶給我們好處也可以帶給我們意想不到的壞處，所以我們要盡量避免AI會帶給我們的壞處，這個技術尚未很純熟，所以我們要好好拿捏，不能完全依賴。
